src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -260,16 +268,16 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
    */
   private def createStore: Future[Map[Call, ExecutionStatus.Value]] = {
     dataAccess.getExecutionStatuses(workflow.id) map { statuses =>
-      workflow.namespace.calls.map {call =>
+      workflow.namespace.workflow.calls.map {call =>
         call -> statuses.get(call.fullyQualifiedName).get}.toMap
     }
   }
 
-  private def buildSymbolStoreEntries(namespace: WdlNamespace, inputs: HostInputs): Traversable[SymbolStoreEntry] = {
+  private def buildSymbolStoreEntries(namespace: NamespaceWithWorkflow, inputs: HostInputs): Traversable[SymbolStoreEntry] = {
     val inputSymbols = inputs.map {case (name, value) => SymbolStoreEntry(name, value, input = true)}
 
     val callSymbols = for {
-      workflow <- namespace.workflows
+      workflow <- namespace.allWorkflows // FIXME: This is the *one* place we're using this. Should we curtail this until we support imported workflows?
```

Comment:
Thinking about this a bit more I think not only should this be changed but we should remove the concept of the imported workflows from `WdlNamespace` altogether as we don't even support them at the moment and it's unclear how it'll work. 

From discussions the other day it sounds like the most likely avenue of support for imported workflows will to be as calls so I think it'd be better to build the logic of call-as-more-than-just-tasks there than to have this chunk of workflows dragging along?